extensions/integration-tests/src/testConfig.ts

```diff
@@ -59,29 +61,29 @@ export class TestServerProfile {
 var TestingServers: TestServerProfile[] = [
 	new TestServerProfile(
 		{
-			serverName: process.env.STANDALONE_SQL,
-			userName: process.env.STANDALONE_SQL_USERNAME,
-			password: process.env.STANDALONE_SQL_PWD,
+			serverName: getEnvironmentVariable(EnvironmentVariable_STANDALONE_SERVER),
+			userName: getEnvironmentVariable(EnvironmentVariable_STANDALONE_USERNAME),
+			password: getEnvironmentVariable(EnvironmentVariable_STANDALONE_PASSWORD),
 			authenticationType: AuthenticationType.SqlLogin,
 			database: 'master',
 			provider: ConnectionProvider.SQLServer,
 			version: '2017'
```

Comment:
I'd get rid of this and replace it with something like 'ServerType' which is set to 'BDC' | 'Standalone' | 'Azure'. It looks to only be used for filtering out the types right now and having the version here could easily be a lie if we update the actual backend server version.

Reply:
I'd keep it this way since we might need to do version specific testing.

Reply:
Having the version could potentially serve useful - but as mentioned currently it's *only* used for filtering the server types anyways so I'd just replace that part of the functionality with something more aligned with what it actually is being used for. There's no reason to keep this around since not only is it confusing with what it's being used for but also could easily be a lie since nothing is preventing us from updating the backend server and not updating this (that could be made "safer" by using something like STANDALONE_2017_* for the env vars).